RecoMTD/TimingIDTools/plugins/TOFPIDProducer.cc

```diff
@@ -149,13 +144,13 @@ void TOFPIDProducer::produce(edm::Event& ev, const edm::EventSetup& es) {
   ev.getByToken(sigmatmtdToken_, sigmatmtdH);
   const auto& sigmatmtdIn = *sigmatmtdH;
 
-  edm::Handle<edm::ValueMap<float>> pathLengthH;
-  ev.getByToken(pathLengthToken_, pathLengthH);
-  const auto& pathLengthIn = *pathLengthH;
+  edm::Handle<edm::ValueMap<float>> tofkH;
+  ev.getByToken(tofkToken_, tofkH);
+  const auto& tofkIn = *tofkH;
```

Comment:
```suggestion
  const auto& tofkIn = ev.get(tofkToken_);
```
is shorter and is a more preferred way to access data (same below; updates to the older code can be done as well, but not required for this review)

Reply:
I agree, an opportunity to clean everything